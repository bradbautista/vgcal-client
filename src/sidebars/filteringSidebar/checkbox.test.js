import React from 'react';
import ReactDOM from 'react-dom';
import Checkbox from './checkbox';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

it('Renders without crashing', () => {
  const div = document.createElement('div');
  const platform = 'PlayStation'
  ReactDOM.render(<Checkbox key={platform} platform={platform}/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Renders the checkbox as expected', () => {
    const platform = 'PlayStation'
    const tree = renderer
        .create(
            <BrowserRouter>
            <Checkbox key={platform} platform={platform}/>
            </BrowserRouter>
            )
        .toJSON();
    expect(tree).toMatchSnapshot();
        
})