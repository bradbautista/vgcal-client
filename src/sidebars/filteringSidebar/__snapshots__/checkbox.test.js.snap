// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the checkbox as expected 1`] = `
<label
  className="checkbox-wrapper"
  htmlFor="PlayStation"
>
  <input
    checked={false}
    id="PlayStation"
    name="PlayStation"
    onChange={[Function]}
    type="checkbox"
    value="PlayStation"
  />
  <span
    className="checkmark"
  />
  PlayStation
</label>
`;
